docs/concepts/rfai.md

```diff
@@ -32,61 +32,65 @@ page_nav:
+* All submissions are evaluated by the SingularityNet foundation to ensure that the acceptance criteria as specified in the request is met and the problem is solved for.
+* The submission should include the github repo of your code
+* The submission should be signed using the same address used to publish the service. This is an important step to ensure that you are the owner of the service.
+* The submission is evaluated as follows
+*  The specified acceptance criteria in the request must be met
+    *  Any performance metrics specified against provided test datasets should be met
+    * Submission should be a service deployed on the SingularityNet platform
 
-## Voting a Solution
 
-* Foundation will vote to shortlist the submissions;
-* Backers can vote to any submission not only to shortlisted ones;
-* Right now only backers in the respective request can vote for the submission;
-* Validation of the solution will be performed offline. Solution will be available as part of Marketplace.
+## Voting for a Solution
 
-## Claims
+  * Foundation will vote to shortlist the submissions
+  * Users backing the service request can vote for any submission (not just the shortlisted ones)
+  * Only the users who are backing a service request can vote for the submission
+  * Validation of the solution will be performed offline.
 
-* Claims can be done only after the evaluation;
-* Solution submitter can claim any time before expiry of the request;
-* Claims will be calculated based on the votes from backers;
-* In case if there is no votes from backers, foundation votes are considered;
-* Claims will be distributed based on the number of votes either by backers or by foundation members;
-* Foundation member pledged amount will be distributed as per the respective member shortlisting.
+## Claiming tokens for a submission
 
-## Reclaim Pledge
+  * Claims can be done only after the evaluation period and before the expiry period
+  * Solution submitter can claim any time before expiry of the request
+  * Claims will be caluculated based on the votes from backing users
+      * In case if there is no votes from any backing user, foundation votes are considered
+  * Claims will be distributed based on the number of votes either by backing users or by foundation members
 
-* Backers can reclaim the pledged amount only when the request is expired;
-* No solution submitter can claim when the request is expired.
+## Reclaiming Tokens
+  * AGI tokens used to back a service request can be reclaimed by the backer after the service request is expired
 
-## Close Request
+## Closing a Request
 
-* Only foundation can forcible close the request even after the approval;
-* Pledges will be given back to the respective backers of the request;
-* Request status will changed to Closed.
+  * Only fondation can close a request.
+  * AGI Tokens used for backing will be returned to the backers
+  * Request status will change to Closed
```

Comment:
put "Closed" in speech marks if you want it capitalised, or change to lowercase.

Reply:
Changed it to lowercase